controllers/remote/cluster_cache_healthcheck_test.go

```diff
@@ -146,6 +146,32 @@ func TestClusterCacheHealthCheck(t *testing.T) {
 			}, 5*time.Second, 1*time.Second).Should(BeTrue())
 		})
 
+		t.Run("during creation of a new cluster accessor", func(t *testing.T) {
+			g := NewWithT(t)
+			ns := setup(t, g)
+			defer teardown(t, g, ns)
+			// Create a context with a timeout to cancel the healthcheck after some time
+			ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
I'm not sure if we have to wait 5 seconds to confirm that the requeue works. Looks like in other cases it would:
* return immediately  (l.548)
* poll forever as well (l.571)

I think if we want to prove that we are in exactly this requeue vs. just polling successfully we have to use an invalid path so the health check would fail.

WDYT?

(If we want to do this, I would reduce the unhealthyThreshold the interval and the requestTime so we don't have to wait 5 seconds. It's always better if unit tests are as quick as possible (~1 sec in this case would be fine)

Reply:
Changed it to one second :-)

Reply:
Thx!

Reply:
P.S. Was mostly commenting as I prefer to keep unit test duration as low as we can